dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOption.java

```diff
@@ -47,14 +80,14 @@ static Builder builder() {
 
   @AutoValue.Builder
   abstract static class Builder {
-    abstract Builder setBomCoordinate(@Nullable String value);
-    abstract Builder setMavenCoordinates(List<String> value);
-    abstract Builder setJarFileList(List<Path> value);
+    abstract Builder setBomCoordinate(@Nullable String coordinate);
+    abstract Builder setMavenCoordinates(List<String> coordinates);
+    abstract Builder setJarFileList(List<Path> paths);
```

Comment:
should probably be nullable

Reply:
As discussed yesterday, not using nullable for now.

Reply:
You can make your `builder()` method above prime everything with empty lists:

```
builder() {
  return new AutoValue_StaticLinkageCheckOption.Builder()
      .setMavenCoordinatesList(ImmutableList.of())
      .setJarFileList(ImmutableList.of());
}
```

Reply:
Updated the `builder()`.